fusedrug/data/protein/structure/extract_chains_to_pdbs.py

```diff
@@ -0,0 +1,69 @@
+from jsonargparse import CLI
+from fusedrug.data.protein.structure.structure_io import (
+    load_pdb_chain_features,
+    save_structure_file,
+)
+from typing import Optional
+
+
+def main(
```

Comment:
what is this script for? please explain in the comments section.

Reply:
added this desc in the docstring:

"Takes an input PDB files and splits it into separate files, one per describe chain, allowing to rename the chains if desired"